src/scene/material.js

```diff
@@ -333,7 +333,9 @@ pc.extend(pc, function () {
      * @param {Number|Array|pc.Texture} data The value for the specified parameter.
      * @author Will Eastcott
```

Comment:
need @param about passFlags.

Reply:
This one is slightly experimental and may change in the future. But if it'll stick, I'll document it.